### ISSUE TITLE
F2 fails headings if there is a <strong> tag inside them.

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#104]
Came across code like this

`<h3><strong>Heading text</strong></h3>`

Sure, its redundant code and not great styling but I don't think it should get a WCAG failure since AT is unaffected. Suggest sniffing if the strong tag is inside a heading and exempting those.

**Comment by irwink** [comment]
I am able to reproduce the error message, however the HTML code was not as simple as the one you provided.  I had to include a div tag inside the header.
`<h3><div><strong>Heading text</strong></div></h3>`

Without the div the tool does not raise an error.  Anyway I'll correct the code to see if there is an open heading tag in the tag stack (and suppress the error).  You would still get a validation error due to a div nested inside a heading tag.

**Comment by irwink** [comment]
Corrected in version 6.9.0.
